tests/test_actions.py

```diff
@@ -599,14 +602,14 @@ def test_implements_protocol(self) -> None:
         assert isinstance(s, Describable)
 
     def test_raises_exception(self) -> None:
-        with pytest.raises(TypeError):
-            SeeAllOf(FakeQuestion())  # type: ignore
+        with pytest.raises(UnableToAct):
+            SeeAllOf(FakeQuestion())
```

Comment:
You see that correctly, we don't actually need a `type: ignore` on this line!

... although it does break up the nice parallelism with the below lines.

Reply:
cool. That must be another mypy update to understand mock objects.